docs/main.js

```diff
@@ -12415,9 +12413,11 @@ var _typeof2 = typeof Symbol === "function" && typeof Symbol.iterator === "symbo
             }, {
               key: 'setProperty',
               value: function setProperty(prop, value) {
-                return this.setModel(Object.assign({}, this.state.model, _defineProperty({}, prop, Object.assign({}, this.state.model[prop], {
+                var newModel = this.setModel(Object.assign({}, this.state.model, _defineProperty({}, prop, Object.assign({}, this.state.model[prop], {
```

Comment:
its seems coming from a previous fix.
https://github.com/Bynder/react-formulation/pull/19/files

I checked the pull request and the main.js it wasn't updated with the new build.